### PR TITLE
FAQ page

### PR DESCRIPTION
**Pull request by francescanarea**
Page for FAQ linked under "Company" footer on urbit.org's homepage.

### CODE REVIEW

**Comment by flowerornament** [comment]
Noticed a commented-out h4 here:

Screenshot 2019-07-11 17 42 57

**Review by flowerornament — CHANGES_REQUESTED** [review]
Please change internal links to "internal link" format. See: https://www.getzola.org/documentation/content/linking/. Basically `@/section/page.md`.

**Comment by flowerornament** [comment]
@alexmatz @francescanarea when are we hoping to merge this?

**Comment by alexmatz** [comment]
as soon as possible @msutherl

**Comment by Fang-** [comment]
It's not liking the newlines for the table of contents.

image

Also feel like we should feature this more prominently than just a tiny link at the bottom of the home page...

**Comment by francescanarea** [comment]
> as soon as possible @msutherl

@alexmatz 
should we then link to the test grant site instead of the currently dead grant.urbit.org site?

**Comment by francescanarea** [comment]
@jyng is making some indigo edits that will allow for universal formatting of drop down lists via details and summary tags

**Comment by alexmatz** [comment]
> Also feel like we should feature this more prominently than just a tiny link at the bottom of the home page...

Agree @Fang-, will make sure it gets a proper place on the next version of the site.

**Review by jfranklin9000 — COMMENTED** [review]
In the `### How do I safely store my stars? {#grants-13}` section is some duplicated text:
"in its own in its own"

File: content/faq.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+The [Using](@/docs/using/_index.md) section of the documentation will help you out.
+
+### How do over-the-air updates work? {#arvo-7}
+
+Your sponsor, a star or galaxy that your ship “lives” under, may send you new source code for your system. When you receive new source code from your sponsor, your system recompiles itself using that code, performs any necessary data migrations, and keeps running. Ideally this happens seamlessly without the user even noticing, although there is sometimes a slowdown while rebuilding the system from source.
+
+### What is Landscape? {#arvo-8}
+
+Landscape is the Arvo web interface that includes social functions such as chat and message boards. You can access Landscape by navigating to <your-planet>.arvo.network, where <your-planet> is your running ship that’s connected to the Arvo network.
+
```
Comment (matildepark) [inline content/faq.md:293]: So, an open question: Isn't Landscape about to be retired in favour of Chat and Publish? Like, very, very soon? Or are we still calling those things Landscape in Modulo?

Comment (flowerornament) [reply content/faq.md:293]: That's correct. We should probably remove stuff about Landscape from this.

**Comment by francescanarea** [comment]
> @jyng is making some indigo edits that will allow for universal formatting of drop down lists via details and summary tags

@jyng do you still think this is necessary? let me know.  wasn't sure what you were intending to do

**Comment by alexmatz** [comment]
@jyng is indigo work related to drop-downs blocking this?

**Comment by alexmatz** [comment]
@msutherl let's merge as soon as the grant website dns updates. the site will live at grants.urbit.org.  i will keep you posted! let's link to the real site url.

**Comment by alexmatz** [comment]
@msutherl @francescanarea the grants.urbit.org is live if you'd like to push this with the proper urls.

**Comment by alexmatz** [comment]
@msutherl anything else required of me to get this out?

**Comment by flowerornament** [comment]
@alexmatz just took a closer look at the copy and I think it needs an editing pass

**Review by flowerornament — APPROVED** [review]

**Pull request merged**